packages/client/src/themes/utilities.css

```diff
@@ -1 +1,42 @@
-@import "tailwindcss/utilities";
\ No newline at end of file
+@import "tailwindcss/utilities";
+
+.table-size-xs {
+  height: calc(100vh - 436px);
+}
+
+.table-size-sm {
+  height: calc(100vh - 418px);
+}
+
+.table-size-md {
+  height: calc(100vh - 402px);
+}
+
+.table-size-lg {
+  height: calc(100vh - 386px);
+}
+
+.table-size-xl {
+  height: calc(100vh - 318px);
+}
```

Comment:
we should ask about this to the designers what the appropriate table size is
if there is height, we can simply make a record object of height name and pixels

Reply:
i make work it with extending heigh theme configuration this is cleaner

```typescript
  theme: {
    extend: {
      height: {
        'table-size-xs': 'calc(100vh - 436px)',
        'table-size-sm': 'calc(100vh - 418px)',
        'table-size-md': 'calc(100vh - 402px)',
        'table-size-lg': 'calc(100vh - 386px)',
        'table-size-xl': 'calc(100vh - 318px)',
        'table-size-full': '100%',
      },
```

```typescript
const TABLE_HEIGHT_SIZES = {
  xs: 'h-table-size-xs',
  sm: 'h-table-size-sm',
  md: 'h-table-size-md',
  lg: 'h-table-size-lg',
  xl: 'h-table-size-xl',
  full: 'h-table-size-full'
}
```